db/migrations/postgresql/_init/V1.0.0.2__schemas.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+-- SCHEMAS WILL BE CREATED INSIDE OF THE CURRENTLY UED DATABASE;
+CREATE SCHEMA IF NOT EXISTS "app";
```

Comment:
since we are now sort of re-creating the migrations, isn't it a good idea to split them per module? These modules will eventually be moved to different repos. We cannot keep all migrations in a single folder. It also doesn't make much sense since ideally deploy 1 module can potentially change only that module's database and not all others

Reply:
We can leave all tables in the public schema, this will help us split them in individual databases after that. At the moment we don't have or need any cross schema queries

Reply:
Ideally each module should have its own database that is not accessed by others. If we change the migrations to PostgreSQL now, then later we need to re-write the migrations yet another time to split the databases. Not sure if that's worth it

Reply:
I agree that splitting those per module would be ideal, it offers separation.

Reply:
This is the schema for the contact module, right? I suggest we rename it to `contacts`, makes it more clear what it's related to.
```suggestion
CREATE SCHEMA IF NOT EXISTS "campaigns";
```

Reply:
If we're going to split databases it makes no sense to split also by schemas. 

It's not hard to send everything to `public` schema and send the module migration files to their own db once we have that.

Reply:
I spent some time reading about what is best - 1 db and multiple schemas or multiple dbs. There were mixed answers but I am convinced that we can make 1 db with multiple schemas - 1 per module and everything will work out just fine. I will implement that for the contact module and add it to its own repo

Reply:
One db, multiple schemas:
- will ease the backup and restore procedures
- good separation
- if we need to do multi-schema reports - will be possible
- running one db instance with one db

Multiple dbs in single instance:
- true separation
- somewhat easy backup/restore procedures
- cross-db queries are possible (via `postgres_fdw`), but not advisable

Multiple dbs in separate instances:
- true separation
- backup separately, restore separately
- no cross-db queries

I'm okay we continue with approach :one: as long as it fits the needss of separation.

If we decide to go :two: or :three: , then it makes sense to use the `public` schema.

Reply:
If we go with 2 or 3 we will be very limited in future. We neeed to design everything very well so we wont have the need of any db queries